static/css/around-map.css

```diff
@@ -35,13 +35,18 @@
   align-self: flex-end;
 }
 
+.selected,
+.marker:hover {
+  border: 4px solid #3BA777!important;
```

Comment:
I'm pretty sure `!important` should be avoided as hell.
When you reach that point it means that the css reached an unbearable architecture.